contracts/TracerPerpetualSwaps.sol

```diff
@@ -73,7 +76,8 @@ contract TracerPerpetualSwaps is
         address _gasPriceOracle,
         uint256 _maxLeverage,
         uint256 _fundingRateSensitivity,
-        uint256 _feeRate
+        uint256 _feeRate,
```

Comment:
The natspec comment for this should be a bit more clear IMO.

It is vague one what exactly `feeRate` is. e.g. it doesn't specify that it is a percentage, times 10^18. e.g. `50*10^18 == 50% fee`. But based on what line 212 seems to say, it should be more like `0.5*10^18 == 50% fee`

or just some way to describe explicitly what it should be

Reply:
Done so now

Reply:
I can't see it